nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/AMXConfigLoader.java

```diff
@@ -170,12 +172,11 @@ private void sortAndDispatch(
                 // ADD: a new ConfigBean was added
                 final ConfigBeanProxy cbp = (ConfigBeanProxy) newValue;
                 final ConfigBean cb = asConfigBean(Dom.unwrap(cbp));
-                final boolean doWait = amxIsRunning();
-                if (handleConfigBean(cb, doWait)) // wait until registered
-                {
+                // wait until registered
+                if (handleConfigBean(cb, true)) {
```

Comment:
Seems to me that this method is now always called with true, so maybe remove the second parameter completely?

Reply:
Well, we are both blind, there is one usage with false :)